docs/content/getting-started/tuples-api-best-practices.mdx

```diff
@@ -19,7 +19,8 @@ import {
 The following list outlines some guidelines and best practices for using <ProductName format={ProductNameFormat.ShortForm}/>:
 
 - Do not store Personal Identifiable Information in tuples
-- Always specify authorization model ID whenever possible
+- Always specify authorization model ID in requests
+- Specify the desired consistency in requests
```

Comment:
Can we link "consistency" to the relevant part in the doc? I would imagine people might not know what that is.

Reply:
This bullet list is the titles of the sections below. I added the link to what consistency is down below 👍